README.md

```diff
@@ -78,26 +78,16 @@ npm run build
 
 Install the socket server node packages. Open a new terminal, `cd` to `cla_frontend` and run:
```

Comment:
I think you've removed the wrong one! Your one, that you thought was right for Windows, is in fact right. The one that was there before has you `cd`ing to `cla_frontend` too many times.

Thinking about it, it makes little to no sense to do the two `cd`s separately, so I'm gonna suggest this change instead:
Remove the ", `cd` to `cla_frontend`" so it just reads "Open a new terminal and run:" and then both levels of cd are done in the code block